.github/workflows/mutation-tests.yml

```diff
@@ -65,14 +65,16 @@ jobs:
           vendor/bin/roave-infection-static-analysis-plugin \
             --ignore-msi-with-no-mutations \
             --git-diff-filter=AM \
-            --git-diff-base=origin/${{ github.base_ref }}
+            --git-diff-base=origin/${{ github.base_ref }} \
+            --min-covered-msi=100 \
+            --min-msi=97
         env:
           INFECTION_BADGE_API_KEY: ${{ secrets.INFECTION_BADGE_API_KEY }}
           STRYKER_DASHBOARD_API_KEY: ${{ secrets.STRYKER_DASHBOARD_API_KEY }}
 
       - name: "Infection"
         if: ${{ github.base_ref == '' }}
-        run: vendor/bin/roave-infection-static-analysis-plugin --ignore-msi-with-no-mutations
+        run: vendor/bin/roave-infection-static-analysis-plugin --ignore-msi-with-no-mutations --min-covered-msi=100 --min-msi=97
```

Comment:
These can/should be set in infection.json.dist (that way, you get same results also locally)